qemu/tests/hotplug_mem_repeat.py

```diff
@@ -45,11 +45,18 @@ def start_test(self):
             arg = "movable_node"
             utils_test.update_boot_option(vm, args_added=arg)
         original_mem = self.get_guest_total_mem(vm)
+
+        rhel_version = "\"Red Hat Enterprise Linux Server release 7\""
+        cmd = "grep %s /etc/redhat-release" % rhel_version
+        status, output = session.cmd_status_output(cmd)
+
+        self.test.log.debug("CMD output: %s" % str(output))
+
         if self.params["test_type"] == "scalability_test":
             error_context.context("Repeat hotplug memory for %s times"
                                   % times, LOG_JOB.info)
             self.repeat_hotplug(vm, target_mems)
-            if self.params.get('os_type') == 'linux':
+            if self.params.get('os_type') == 'linux' and status != 0:
```

Comment:
Instead of 'status', prefer a clearer and meaningful variant name.

Reply:
What about something like: **rhel_version_cmd_status**? according also to previous variable **rhel_version**

Reply:
How about keep `status` in L51 and introduce a new one, like `is_rhel_7` ?

Reply:
Updated